test/compiler.js

```diff
@@ -27,7 +27,7 @@ const compilerPackage = require('../');
 const Compiler = compilerPackage.compiler;
 const packageInfo = require('../package.json');
 const Semver = require('semver');
-const compilerVersionExpr = /^Version:\sv(.*)$/m;
+const compilerVersionExpr = /^Version: v(\d+)(?:[-\.][-a-z0-9]+)?$/m;
```

Comment:
If this is useful both here and in build_compiler, maybe move it somewhere common.